manifest.webapp

```diff
@@ -24,6 +24,11 @@
       "type": "io.cozy.apps",
       "verbs": ["GET", "POST", "PUT"]
     },
+    "settings": {
+      "description": "Required by the cozy-bar to know which applications are coming soon",
```

Comment:
I would prefer to explain say that is about the cozy context since it won't be used for coming soon apps only (but also Claudy)